src/ctap/key_material.rs

```diff
@@ -14,6 +14,9 @@
 
 pub const ATTESTATION_PRIVATE_KEY_LENGTH: usize = 32;
 pub const AAGUID_LENGTH: usize = 16;
+pub const _UPGRADE_PUBLIC_KEY_LENGTH: usize = 77;
 
 pub const AAGUID: &[u8; AAGUID_LENGTH] =
     include_bytes!(concat!(env!("OUT_DIR"), "/opensk_aaguid.bin"));
+pub const _UPGRADE_PUBLIC_KEY: &[u8; _UPGRADE_PUBLIC_KEY_LENGTH] =
```

Comment:
Last time, after  discussion, I realized that `#[warn(dead_code)]` (and not `#[allow(dead_code)]`) might be more convenient (no need to change names). As you want.

Reply:
I tried that, and realized that we error on warning. So it's either `allow` or renaming. And in this case, it wasn't too much work to rename.

Reply:
Oh right this is not great. We might want to remove that rule in `develop` branch.